src/components/VmDetails/cards/DisksCard/index.js

```diff
@@ -130,7 +130,7 @@ class DisksCard extends React.Component {
         idPrefix={idPrefix}
         icon={{ type: 'pf', name: 'storage-domain' }}
         title={msg.disks()}
-        editTooltip={msg.disksCardEditTooltip({ vmName: vm.get('name') })}
+        editTooltip={msg.edit()}
```

Comment:
Do we really want to see only "Edit" instead of 'Edit Disks for {vmName}'? I think this change is unnecessary.

Reply:
Another thing is that you are removing `disksCardEditTooltip` message here and that makes `disksCardEditTooltip` useless, but I still can see it in the _messages.js_ [file](https://github.com/oVirt/ovirt-web-ui/pull/1289/files#diff-c7c4c4da7b77958d3e42b7c152709bffR220). Is there any reason for that?

Reply:
> Do we really want to see only "Edit" instead of 'Edit Disks for {vmName}'? I think this change is unnecessary.
I stuck to the document:
https://docs.google.com/document/d/1ouQ7b4tKH1ksmB54hnPz69rkHcynv7qhG3hDXf342XA/edit?disco=AAAAKMHErgg
If there are any other instructions I'll change it accordingly

Regarding:
> Another thing is that you are removing `disksCardEditTooltip` message here and that makes `disksCardEditTooltip` useless, but I still can see it in the _messages.js_ [file](https://github.com/oVirt/ovirt-web-ui/pull/1289/files#diff-c7c4c4da7b77958d3e42b7c152709bffR220). Is there any reason for that?

I'll remove it thanks :)